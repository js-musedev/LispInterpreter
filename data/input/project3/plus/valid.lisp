(PLUS 8 4)
(PLUS (PLUS 4 3) (PLUS (PLUS 1 2) 4))
(PLUS (COND ((GREATER 3 45) 1) (T 2)) 2854)