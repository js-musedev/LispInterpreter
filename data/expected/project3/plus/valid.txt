12
14
2856
